tests/xharness/Jenkins/TestVariationsFactory.cs

```diff
@@ -225,9 +225,9 @@ IEnumerable<TestData> GetTestData (RunTestTask test)
 						var canSymlink = task.Platform.CanSymlink ();
 
 						if (!string.IsNullOrEmpty (mtouch_extra_args))
-							clone.Xml.AddExtraMtouchArgs (mtouch_extra_args, task.ProjectPlatform, configuration);
+							clone.Xml.AppendExtraMtouchArgs (mtouch_extra_args);
 						if (!string.IsNullOrEmpty (bundling_extra_args))
-							clone.Xml.AddMonoBundlingExtraArgs (bundling_extra_args, task.ProjectPlatform, configuration);
```

Comment:
What happened to `AddExtraMtouchArgs` and `AddMonoBundlingExtraArgs`? I don't see them deleted in the changes.

Reply:
They're not defined in this repo, they come from dotnet/xharness:

https://github.com/dotnet/xharness/blob/7501abf49c3db1689f8a5507ff009cc1a5cd9323/src/Microsoft.DotNet.XHarness.iOS.Shared/Utilities/ProjectFileExtensions.cs#L393-L395